Pod/UI/Classes/UILabel+BonMotUtilities.h

```diff
@@ -0,0 +1,27 @@
+//
+//  UILabel+BonMotUtilities.h
+//  BonMot
+//
+//  Created by Nora Trapp on 3/2/16.
+//
+//
+
+@import UIKit;
+@protocol BONChainable;
+
+@interface UILabel (BonMotUtilities)
+
+/**
+ *  Assign a @p BONChainable object to apply to the label text. When a new value is assigned to @p text the chain attributes will be applied.
```

Comment:
#pp use `@c` to denote that these are code, not just reformatted. (they do the same thing in practice, hence #pp)